Frends.AMQP.Send/Frends.AMQP.Send.Tests/TestAmqpBrokerHelper.cs

```diff
@@ -6,133 +6,111 @@
 using System.Net;
 using System.Net.Sockets;
 
-namespace Frends.AMQP.Send.Tests
+namespace Frends.AMQP.Send.Tests;
+
+/// These tests here use a test AMQP server. The test server is started and killed automatically.
+/// https://github.com/Azure/amqpnetlite/tree/master/test/TestAmqpBroker
+/// 
+/// If you want to test this against a real ActiveMQ server, Docker is recommended:
+/// docker pull rmohr/activemq
+/// docker run -p 5672:5672 -p 8161:8161 rmohr/activemq
```

Comment:
Please add `-d` to docker command to run the container in detached mode (in the background).